exercises/05/readme.md

```diff
@@ -223,11 +223,17 @@ curl ^
 
 :point_right: Order 9 copies of Life, The Universe And Everything (specifying an order ID):
 
-```
+```sh
 curl \
   -d '{"ID": "527ef85a-aef2-464b-89f6-6a3ce64f2e14", "book_ID":427,"quantity":9}' \
   -H 'Content-Type: application/json' \
   http://localhost:4004/catalog/Orders
+
+# For Windows users:
```

Comment:
better as a separate non-code-formatted paragraph, as above, to keep the invocations separately in two code blocks (same comment for the rest of these same changes)

Reply:
ok